be/src/cache/block_cache.cpp

```diff
@@ -0,0 +1,142 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#include "cache/block_cache.h"
+
+#include <fmt/format.h>
+
+#include "common/config.h"
+#include "common/statusor.h"
+#include "common/logging.h"
+#include "gutil/strings/substitute.h"
+#include "cache/fb_cachelib.h"
+
+//#include "util/hash_util.hpp"
+#include "formats/orc/apache-orc/c++/src/Murmur3.hh"
```

Comment:
We already have murmurhash implementation in SR, pherhaps you can check it. If not, you can put that implementation in SR, instead of refering file in ORC module.  In short word, better not to cross reference between modules

Reply:
> We already have murmurhash implementation in SR, pherhaps you can check it. If not, you can put that implementation in SR, instead of refering file in ORC module. In short word, better not to cross reference between modules

Yes, you are right. I did that firstly but found some global variables conflict with cachelib library, so I choose to use this module directly. As it only used for content validation before, and I'll remove this line.